frontend/src/app/main/ui/workspace/sidebar/options/menus/component.cljs

```diff
@@ -225,6 +230,81 @@
         (when (or editing? creating?)
           [:div {:class (stl/css  :counter)} (str size "/300")])]])))
 
+
+(mf/defc component-variant*
+  {::mf/props :obj}
```

Comment:
Minor: :fire:, not necessary when component follows the `*` naming convention, it is default for that components.